src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -569,7 +569,7 @@ public void SendWorkerWarmupRequest()
             {
                 _workerChannelLogger.LogDebug("Sending WorkerWarmupRequest to WorkerProcess with Pid: '{0}'", _rpcWorkerProcess.Id);
 
-                RegisterCallbackForNextGrpcMessage(MsgType.WorkerWarmupResponse, TimeSpan.FromMinutes(1), 1,
+                RegisterCallbackForNextGrpcMessage(MsgType.WorkerWarmupResponse, TimeSpan.FromMinutes(1.5), 1,
```

Comment:
Are we sure that the worker just needed more time and that there isn't another underlying issue here?

Reply:
Yes, the requests were completed successfully by the worker at a later point by which the timeout occurred

Reply:
The worker warmup takes more than a minute? How do we know 1.5 minute is enough (in future)?